app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int = 0,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, is_hungry=is_hungry, appetite=7)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(list_animals: list[Animal]) -> int:
+    sum_food = sum([
+        animal.appetite for animal in list_animals
+        if animal.is_hungry
+    ])
+    for animal in list_animals:
+        animal.feed()
+    return sum_food
```

Comment:
There is no need to separately calculate the `sum_food`, it will be a collective output of the `feed` method from all animals. Do not create additional variables, perform `sum` in a `return` statement. You should pass a list comprehension (or a generator expression preferably) to a `sum` that will call `feed` method on all animals from `list_animals`.